### ISSUE TITLE
Unable to import Java classes: Illegal char<:> at index 4

### ISSUE THREAD

**Issue by kauereinbold** [apache/incubator-kie-tools#1272]
**Describe the Bug**

I am unable to import java classes. When typing the name of the class in the search box, no class appear.
In Developer tools I see following error:

```
dmn-loader.js:2 Error: Illegal char <:> at index 4: file://D:\src\drools\kogito-examples\kogito-quarkus-examples\dmn-quarkus-example\src\main\java\com\mc\Activator.java
    at e.callback (DmnEditorEnvelopeApp.js:2:517048)
    at e.receive (DmnEditorEnvelopeApp.js:2:518804)
    at Object.receive (DmnEditorEnvelopeApp.js:2:514121)
    at e.receive (DmnEditorEnvelopeApp.js:2:521333)
    at eventListener.eventListener (DmnEditorEnvelopeApp.js:2:520493)
```

I was following this blog post entry https://blog.kie.org/2022/05/dmn-types-from-java-classes.html. I tried creating my own project and also cloned examples repository. Same issue on both.

**Steps to Reproduce**

1. open https://github.com/kiegroup/kogito-examples/tree/stable/kogito-quarkus-examples/dmn-quarkus-example in VS Code
2. create Activator class:

```
package com;

import org.kie.api.project.KieActivator;

@KieActivator
public class Activator {
    
}
```

3. create java bean:

```
package com;

public class Person {
    String firstName;
    public String getFirstName() {
        return firstName;
    }
    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }
}
```

4. Open Traffic Violation.dmn and try to search the Person class
5. In Developer tools Error: Illegal char <:> at index 4: error appears.

**Expected Behavior**

Plugin should be able to fetch my model classes.

Environment

- OS: Windows
- Visual Studio Code version: 1.71.2
- DMN Editor plugin: 0.23.0
- Language Support for Java(TM) by Red Hat v1.11.0

Obs: Copy from the bug found in another [repository](https://github.com/bpmn-io/dmn-js/issues/703).

**Comment by yesamer** [comment]
HI @KaueReinbold. 
Thank you for reporting that.
Most likely, this is a regression introduced in version 0.23.0. Please revert the extension to version 0.22.0 to verify. In the meantime, I'll provide a fix.

**Comment by kauereinbold** [comment]
Hello @yesamer,
Thank you for the response, I tried previous versions until v0.19.0, and the problem still happens.
I created a [repository](https://github.com/KaueReinbold/dmn-quarkus-example) with the code I am using if I am missing anything by any chance.

**Comment by yesamer** [comment]
@KaueReinbold A fix is on its way for the next version (`0.25`).

**Comment by yesamer** [comment]
https://issues.redhat.com/browse/KOGITO-8119

**Comment by yesamer** [comment]
@KaueReinbold PR merged. The fix will be available in `0.26.0` version. Unfortunately, I missed the `0.25.0` release.

**Comment by kauereinbold** [comment]
Thank you @yesamer ! I am looking forward for the update.

**Comment by yesamer** [comment]
Closing as Resolved - @KaueReinbold please feel free to reopen if the fix is not applied in `0.26.0` as expected.
